Composer/packages/client/src/components/Orchestrator/OrchestratorForSkillsDialog.tsx

```diff
@@ -41,25 +47,14 @@ export const OrchestratorForSkillsDialog = () => {
     setShowOrchestratorDialog(false);
   };
 
-  const setVisibility = () => {
-    if (showOrchestratorDialog) {
-      if (hasOrchestrator) {
-        setShowOrchestratorDialog(false);
-        return false;
-      }
-      return true;
-    }
-    return false;
-  };
-
-  const onDismissHandler = (event: React.MouseEvent<HTMLButtonElement> | undefined) => {
+  const onDismissHandler = () => {
     setShowOrchestratorDialog(false);
   };
 
   return (
     <DialogWrapper
       dialogType={DialogTypes.CreateFlow}
-      isOpen={setVisibility()}
+      isOpen={showOrchestratorDialog}
```

Comment:
does this ever evaluate to true?? all I see in this file is `setShowOrchestratorDialog(false)`

Reply:
nevermind, I see now that it's recoil state and not react